src/public/auth.html

```diff
@@ -122,14 +127,25 @@ <h1 class="pb-4">Azure Static Web Apps Auth</h1>
               <div class="form-group row">
                 <label for="userRoles" class="col-4 col-form-label">User’s roles</label>
                 <div class="col-8">
-                  <textarea id="userRoles" name="userRoles" cols="40" rows="5" class="form-control" aria-describedby="userRolesHelpBlock"></textarea>
+                  <textarea id="userRoles" name="userRoles" cols="40" rows="3" class="form-control" aria-describedby="userRolesHelpBlock"></textarea>
                   <small id="userRolesHelpBlock" class="form-text text-muted">Roles used during authorization. One role per line.</small>
                   <small id="userRolesHelpBlock" class="form-text text-muted"
                     >Note: roles "authenticated" and "anonymous" will be added automatically if not provided.</small
                   >
                 </div>
               </div>
 
+              <div class="form-group row">
```

Comment:
Perhaps in future this input should be swapped out for a way of adding key/value pairs for claims. That way we ensure users have claims with properties typ and val, and don't have to worry about malformed JSON.